wasm_utils/src/api_serialization/get_links.rs

```diff
@@ -20,3 +20,11 @@ impl GetLinksResult {
         &self.addresses
     }
 }
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GetLinksLoadElement<T> {
+    pub address: Address,
```

Comment:
I'm not sure we need the `address` field because every entry has an .address() function that returns the entrie's address.  I guess there could be a santity check that when the link is loaded that the actual address matches the calculated `.address()`

Reply:
@zippy yes i agree, would be a validation/sanity check rather than data, "out of the tarpit" and all :)

Reply:
so it's just return address OR entry, right? an address can be "trade in" for an entry via some workflow that relies on network/storage but an entry should always dictate its own address deterministically

Reply:
Cool I actually didn't know you could just call `.address()` on an entry.

So this makes sense for `get_links_and_load` but I think `get_links_and_load_as` still needs to return the struct with a separate address field. This is because the result is no longer an `entry` type but a native struct type. What do you think?